src/js/views/forms/form/form_views.js

```diff
@@ -50,10 +51,10 @@ const ContextTrailView = View.extend({
     Radio.request('history', 'go:back');
   },
   templateContext() {
-    const isSidebarVisible = !!this.model.get('sidebar');
+    const isSidebarVisible = !this.model.get('isExpanded');
```

Comment:
let's replace `isSidebarVisible` with `isExpanded` and switch the logic in the template.  And replace the `expanded` below with `isExpanded` as well.